lib/heroku/kensa/check.rb

```diff
@@ -173,6 +173,14 @@ def call!
           response.is_a?(Hash) && response.has_key?("id")
         end
 
+        check "id does not contain heroku_id" do
+          if response["id"] && response["id"].respond_to?(:include?) && response["id"].include?(data["heroku_id"])
```

Comment:
> I took "heroku id" in the TP card to mean the hid.

Sorry, could have been clearer. I have a dataclip somewhere and I saw some IDs were: `heroku_app125123` which is also problematic, so testing for the numeric part is hopefully sufficient.

>  some that return a `{ id: nil }`

Yikes. Our actual provisioning code definitely expects an `id` 100% of the time for provision requests. We should add a test for that if there isn't already.